website/docs/d/cis_filter.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_filter"
+description: |-
+  Get information on an IBM Cloud Internet Services Filters.
+---
+
+# ibm_cis_filter
```

Comment:
should be `ibm_cis_filters`.. Please make this change everywhere in this datasource doc